prototype/mixins.py

```diff
@@ -29,3 +31,34 @@ class SlugFromKwargsMixin():
     @property
     def slug(self):
         return self.kwargs['slug']
+
+
+class InternationalNewsCMSPageMixin(GetCMSPageByFullPathMixin):
+
+    @cached_property
+    def page(self):
+        response = cms_api_client.lookup_by_full_path(
+            full_path=prefix_international_news_url(self.request.path),
+            draft_token=self.request.GET.get('draft_token'),
+        )
+        return handle_cms_response(response)
+
+
+class SocialLinksMixin():
+
+    @cached_property
```

Comment:
`cached_property` is normally reserved for when the function is performing an "expensive" operation such as retrieving data from an API. It speeds up the code at the tradeoff of adding complexity

this function looks rather cheap, so it's hard to justify the added complexity of using `cached_property`

Reply:
okay i wondered whether it was worth it to use that. i can remove it